controllers/dcrclient.go

```diff
@@ -622,7 +622,12 @@ func (w *walletSvrManager) executeInSequence(fn functionName, msg interface{}) i
 
 		if !w.checkForSyncness(spuirs) {
 			log.Infof("StakePoolUserInfo across wallets are not synced.  Attempting to sync now")
-			w.syncTickets(spuirs)
+			err := w.syncTickets(spuirs)
+			if err != nil {
+				log.Errorf("failed to syncTickets: %v", err)
+				resp.err = err
+				return resp
```

Comment:
I do not think we want to quit the `executeInSequence` loop (effectively kill dcrstakepool) if `syncTickets` has an error.  It will have a non-nil error if `rpcclient.(*Client).AddTicket` has an error.  It is not clear what would cause this error, but it could easily be transient connectivity issues, or an already added ticket, etc.

See the docs  for `(*walletSvrManager).StakePoolUserInfo`, the function that gets to this point in `executeInSequence`:

```go
// StakePoolUserInfo gets the voting service user information for a given user.
//
// This can race depending on what wallet is currently processing, so failures
// from this function should NOT cause fatal errors on the web server like the
// other RPC client calls.
func (w *walletSvrManager) StakePoolUserInfo(...
```

Reply:
I suggest we put this as `_ = w.syncTickets(spuirs)` with a comment, or simply log the error (from `AddTicket`).

Reply:
for now, I will just log all the missed errors